lib/src/scss-shared/_nx-focus-border-helpers.scss

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+// For dynamically calculating focus border styles based on height and width
+// Border size is 1px border and 6px here represents total space in each axis
+// between border and control elements. Values are based on RSC design spec.
+
+// NxRadio and NxCheckbox require the left property for correct positioning of focus border
+@mixin nx-radio-checkbox-focus-border($height, $width) {
+    height: calc(#{$height} + 6px);
+    left: -4px;
+    width: calc(#{$width} + 6px);
+}
+
+// NxToggle requires an additional top property for correct positioning of focus border
+@mixin nx-toggle-focus-border($height, $width) {
+    @include nx-radio-checkbox-focus-border($height, $width);
+    top: -4px;
```

Comment:
Since this is specific to the one case I'm not sure it needs to be a second mixin, it could just be directly in the nx-toggle styles

Reply:
13087749db49d32aec38c7cef7040a9bfa4b8ed2